src/lua/api-gateway/util/logger.lua

```diff
@@ -1,18 +1,31 @@
 local set = false
 
-function getLogFormat(level, debugInfo, ...)
-    return level, "[", debugInfo.short_src,
+local function is_in_init_phase()
+    return ngx.var.requestId
+end
+
+local function getLogFormat(level, debugInfo, ...)
+    local status, result = pcall(is_in_init_phase)
+    --- testing for init phase
+    if status then
+        return level, "[", debugInfo.short_src,
```

Comment:
IMO, if status is true then req_id should be result else it should be set to "N/A".